public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchLogsQueryRunner.ts

```diff
@@ -312,7 +310,7 @@ export class CloudWatchLogsQueryRunner extends CloudWatchRequest {
     queryParams: CloudWatchLogsRequest[],
     options?: DataQueryRequest<CloudWatchQuery>
   ): Observable<DataFrame[]> {
-    const range = options?.range || this.timeSrv.timeRange();
+    const range = options?.range || getDefaultTimeRange();
```

Comment:
The startQuery requests are the only ones that actually use the timeRange on the backend, and those always send the range in the options.